controllers/defaults.go

```diff
@@ -0,0 +1,31 @@
+package controllers
+
+// Below are the default constants that we will
+// use throughout the patterns operator code
+const (
+	// Default Operator Namespace
+	OperatorNamespace = "openshift-operators"
+	// Default Operator Config Map File Name
```

Comment:
It's not really a file, can we call it something more obvious? OperatorConfigurationMap or something along those lines?

Reply:
OperatorConfigMap it is.